index.html

```diff
@@ -305,6 +305,48 @@ <h2>Disputes</h2>
       </p>
     </section>
 
+    <section>
+      <h3>Presentations</h3>
+
+      <p>
+<a>Verifiable credentials</a> may be presented to a <a>verifier</a> by
+using a <a>verifiable presentation</a>. A <a>verifiable presentation</a> can
+be targeted to a specific <a>verifier</a> by using a Linked Data Proof
+that includes a <code>domain</code> and <code>challenge</code>. This also
+helps prevent a <a>verifier</a> from reusing a <a>verifiable presentation</a>
+as their own.
+      </p>
+
+      <p>
+The following <a>verifiable presentation</a> is for authenticating to
+a website <code>https://example.com</code>. Note that the <code>domain</code>
+value can be any string or URI and the <code>challenge</code> should be a
+randomly generated string.
+      </p>
+
+
+      <pre class="example nohighlight" title="A targeted verifiable presentation">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": "VerifiablePresentation,
+  "verifiableCredential": { ... },
```

Comment:
needs CSS magic on `...` here and below (and I think `..` in `jws` should be `...`)

Reply:
Possibly yes to the CSS magic ... but the `..` (two dots) is correct... that's actually part of the JWS.

Reply:
CSS magic added.